### PR TITLE
Loadout tweaks + accessory path improvements

### LINKED ISSUE #3542: Scientist can't start with prescription mesons
See title, scientists, especially anomalist or xenoarch would benefit from permission to use that in custom loadout

### PR DESCRIPTION
**Pull request by mistyLuminescence**
- Ties are now /accessory/tie, not /accessory.
- Fixed a typo in railing.dm
- Armbands, waistcoats, holsters, jackets, ties and scarves now come in selection menus (declutters loadout).
- Renamed webbing and drop pouches to black/brown/white in the loadout, since it's not job-restricted to specific departments.
- Psychiatrists can now take medical boots, winter coat, EMT labcoat, and the medical poncho, like the rest of medical. Still can't have MedHUDs from the loadout, though.
- Scientists and Research Directors can now take mesons.
- Scrubs are now no longer limited to medical staff; pregnant Tajaran can take them too.
- Added the bathrobe, bunny slippers, kitty ears to the loadout.
- Only Detectives can take forensic gloves - this has been clarified.
- Cowboy hats are now worth the same as every other hat.
- Prescription Command sunglasses are now worth the same as regular Command sunglasses.

Fixes #3542 by letting Scientists and RD take mesons.

### CODE REVIEW

**Comment by mistyLuminescence** [comment]
stay tuned folks

File: code/modules/client/preference_setup/loadout/loadout_eyes.dm [hunk 1]
```diff
@@ -53,7 +53,7 @@
 /datum/gear/eyes/medical
 	display_name = "Medical HUD (Medical)"
 	path = /obj/item/clothing/glasses/hud/health
-	allowed_roles = list("Medical Doctor","Chief Medical Officer","Chemist","Paramedic","Geneticist")
+	allowed_roles = list("Medical Doctor","Chief Medical Officer","Chemist","Paramedic","Geneticist", "Psychiatrist")
```
Comment (Anewbe) [inline code/modules/client/preference_setup/loadout/loadout_eyes.dm]: I am not particularly a fan of this change.

File: code/modules/client/preference_setup/loadout/loadout_suit.dm [hunk 2]
```diff
@@ -460,4 +460,9 @@
 /datum/gear/suit/snowsuit/cargo
 	display_name = "snowsuit, supply"
 	path = /obj/item/clothing/suit/storage/snowsuit/cargo
-	allowed_roles = list("Quartermaster","shaft Miner","Cargo Technician","Head of Personnel")
\ No newline at end of file
+	allowed_roles = list("Quartermaster","shaft Miner","Cargo Technician","Head of Personnel")
+
+/datum/gear/suit/ianshirt
```
Comment (Anewbe) [inline code/modules/client/preference_setup/loadout/loadout_suit.dm]: Not really feeling this, for the time being.

File: code/modules/client/preference_setup/loadout/loadout_suit.dm [hunk 3]
```diff
@@ -149,7 +149,7 @@
 /datum/gear/suit/labcoat/emt
 	display_name = "labcoat, EMT (Medical)"
 	path = /obj/item/clothing/suit/storage/toggle/labcoat/emt
-	allowed_roles = list("Medical Doctor","Chief Medical Officer","Chemist","Paramedic","Geneticist")
+	allowed_roles = list("Medical Doctor","Chief Medical Officer","Chemist","Paramedic","Geneticist", "Psychiatrist")
```
Comment (Anewbe) [inline code/modules/client/preference_setup/loadout/loadout_suit.dm]: I'd rather we not add this.

File: code/modules/client/preference_setup/loadout/loadout_eyes.dm (hunk 1 above)
Comment (mistyLuminescence) [reply code/modules/client/preference_setup/loadout/loadout_eyes.dm]: If chemists and geneticists (neither of which 'need' one per se) can have it through the loadout, and their job has access to it...

File: code/modules/client/preference_setup/loadout/loadout_suit.dm (hunk 3 above)
Comment (mistyLuminescence) [reply code/modules/client/preference_setup/loadout/loadout_suit.dm]: See above, literally every other med job has access to these at roundstart, and every med job has access to the lockers with them inside.

File: code/modules/client/preference_setup/loadout/loadout_suit.dm (hunk 2 above)
Comment (mistyLuminescence) [reply code/modules/client/preference_setup/loadout/loadout_suit.dm]: Poor Ian. Need me to kill the HoP's dreams?

File: code/modules/client/preference_setup/loadout/loadout_suit.dm (hunk 3 above)
Comment (mistyLuminescence) [reply code/modules/client/preference_setup/loadout/loadout_suit.dm]: Same logic as above, literally every other med job - even geneticist, which I'm not entirely sure we have - gets it, and every med job (inc. psych) has access to the lockers with these in it.

File: code/modules/client/preference_setup/loadout/loadout_suit.dm (hunk 2 above)
Comment (mistyLuminescence) [reply code/modules/client/preference_setup/loadout/loadout_suit.dm]: Poor Ian. Need me to kill the HoP's dreams?

File: code/modules/client/preference_setup/loadout/loadout_eyes.dm (hunk 1 above)
Comment (Anewbe) [reply code/modules/client/preference_setup/loadout/loadout_eyes.dm]: Chemists have a good reason to have a MedHud, as much as I might personally dislike that. Psychiatrists really, really do not.

File: code/modules/client/preference_setup/loadout/loadout_suit.dm (hunk 3 above)
Comment (Anewbe) [reply code/modules/client/preference_setup/loadout/loadout_suit.dm]: Fair, on this point at least. Disregard.

File: code/modules/client/preference_setup/loadout/loadout_suit.dm (hunk 2 above)
Comment (Anewbe) [reply code/modules/client/preference_setup/loadout/loadout_suit.dm]: Yes.

**Comment by Belsima** [comment]
Whats the default accessory cost?

File: code/modules/client/preference_setup/loadout/loadout_shoes.dm [hunk 4]
```diff
@@ -195,6 +195,10 @@
 	..()
 	gear_tweaks = list(gear_tweak_free_color_choice)
 
+/datum/gear/shoes/slippers
```
Comment (Belsima) [inline code/modules/client/preference_setup/loadout/loadout_shoes.dm]: this isn't slipper selection. i feel betrayed.

**Comment by mistyLuminescence** [comment]
Forgot to update default accessory cost, that's why Travis failed I think - also add more slippers for a selection

**Comment by Anewbe** [comment]
You've tested all of this, correct?

File: code/modules/client/preference_setup/loadout/loadout_accessories.dm [hunk 5]
```diff
@@ -1,42 +1,28 @@
 /datum/gear/accessory
-	display_name = "armband, red"
-	path = /obj/item/clothing/accessory/armband
```
Comment (Atermonera) [inline code/modules/client/preference_setup/loadout/loadout_accessories.dm]: code/modules/client/preference_setup/loadout/loadout_accessories.dm:3:error: /obj/item/accessory/: undefined type path

Compile error

Comment (mistyLuminescence) [reply code/modules/client/preference_setup/loadout/loadout_accessories.dm]: it stopped working again save me from this hell

**Comment by mistyLuminescence** [comment]
remaking but _better this time_

**Pull request closed without merging**